File: app/src/main/java/app/fit/fitndflow/data/dto/categories/CategoryDto.java
package app.fit.fitndflow.data.dto.categories;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

import app.fit.fitndflow.data.dto.StringInLanguages;

public class CategoryDto implements Serializable {

    @SerializedName("id")
    public int id;

    @SerializedName("name")
    public StringInLanguages name;

    @SerializedName("exercises")
    public List<ExerciseDto> exerciseDtoList;

}

File: app/src/main/java/app/fit/fitndflow/data/dto/categories/CategoryDto.kt
package app.fit.fitndflow.data.dto.categories

import app.fit.fitndflow.data.dto.StringInLanguagesDto
import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class CategoryDto(@SerializedName("id") var id: Int?, @SerializedName("name") var name: StringInLanguagesDto?, @SerializedName("exercises") var exerciseDtoList: List<ExerciseDto>?): Serializable
